Cargo.toml

```diff
@@ -21,14 +21,14 @@ async-trait = "0.1"
 bitcoin = "0.31"
 candid = { version = "0.10", features = ["value"] }
 clap = { version = "4", features = ["derive"] }
-did = { git = "https://github.com/bitfinity-network/bitfinity-evm-sdk", package = "did", tag = "v0.18.x" }
+did = { git = "https://github.com/bitfinity-network/bitfinity-evm-sdk", package = "did", tag = "v0.18.1" }
 env_logger = { version = "0.11", default-features = false }
-eth-signer = { git = "https://github.com/bitfinity-network/bitfinity-evm-sdk", package = "eth-signer", tag = "v0.18.x" }
-ethereum-json-rpc-client = { git = "https://github.com/bitfinity-network/bitfinity-evm-sdk", package = "ethereum-json-rpc-client", tag = "v0.18.x" }
+eth-signer = { git = "https://github.com/bitfinity-network/bitfinity-evm-sdk", package = "eth-signer", tag = "v0.18.1" }
+ethereum-json-rpc-client = { git = "https://github.com/bitfinity-network/bitfinity-evm-sdk", package = "ethereum-json-rpc-client", tag = "v0.18.1" }
```

Comment:
We came up with new tagging method of using `v.0.0.x` to avoid opening a new PRs and updating the tags manually in other dependency repos; I suggest we retain the some method and if its a major bump; create a new tag `v0.19.x`